.gitignore

```diff
@@ -1 +1,2 @@
 *.qm
```

Comment:
If the translations are built in build/ now, this is no longer needed. I see you removed it in asteroid-settings, please remove it here as well

Reply:
It looks like you may have missed that comment? This is not super duper important but since you're at it, it's a good time to clean this up too ;)